sons-of-the-forest.kvp

```diff
@@ -14,7 +14,7 @@ Meta.MinAMPVersion=2.4.5.0
 Meta.SpecificDockerImage=cubecoders/ampbase:wine8
 Meta.DockerRequired=False
 Meta.ContainerPolicy=RecommendedOnLinux
-Meta.ContainerPolicyReason=The server requires Wine 7 (or later) to run on Linux. Using a container avoids the need to install this dependency on the host.
+Meta.ContainerPolicyReason=The server requires Wine 8 (or later) and Xvfb to run on Linux. Using a container avoids the need to install these dependencies on the host.
```

Comment:
Confused by this change.
He's not using Xvfb though, no?

Or else at this line:
https://github.com/CubeCoders/AMPTemplates/pull/406/files#diff-7ffb7c9260611f98c7e7ecbd202aec34fd1d400c63aa3da5a712162ee652db8fR14

Wouldn't it show:
`cubecoders/ampbase:xvfb*` instead of `cubecoders/ampbase:wine8`

Reply:
It is confusing. The Linux executable calls xvfb-run and the command line args covers Wine. The Wine image contains Xvfb.